task2/js/app.js

```diff
@@ -18,8 +18,8 @@ document.addEventListener('DOMContentLoaded', () => {
         const zombieElement = document.createElement('div');
         zombieElement.classList.add('zombie-item', zombie.icons);
 
-        zombieElement.addEventListener('click', (event) => {
-            event.target.classList.add('zombie-dead');
+        zombieElement.addEventListener('click', function() {
+            this.classList.add('zombie-dead');
```

Comment:
well... this will work exactly in the same way as "event.target"
please implement this task using "zombieElement" variable